packages/rs-drive/src/drive/batch/drive_op_batch/identity.rs

```diff
@@ -49,6 +51,14 @@ pub enum IdentityOperationType {
         disable_at: TimestampMillis,
     },
 
+    /// Re-Enable Identity Keys
+    ReEnableIdentityKeys {
```

Comment:
I think I might have missed the discussion where we decided to make it possible to re-enable keys?